data/pc/1.18/protocol.json

```diff
@@ -4274,6 +4360,15 @@
             }
           ]
         ],
+        "packet_simulation_distance": [
+          "container",
+          [
+            {
+              "name": "dist",
```

Comment:
why not simulationDistance ?

Reply:
Seems redundant and long winded, the packet gives that context

```js
client.on("simulation_distance", (packet) => {
  console.log(packet.dist);
})
```

Reply:
ok but let's at least have a full word: distance